docs/fw_core.yaml

```diff
@@ -1172,15 +1181,7 @@ components:
           type: boolean
         editGroupId:
           type: string
```

Comment:
is removing the areas array intentional?

Reply:
Good catch, not really, stoplight has removed it for some reason, let me amend